workshops/CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to our Workshops
+
+Any contributions to our workshops are _very_ welcome, even from beginners!
+
+## We <3 Questions
+
+Before we get started, if you ever have any questions about anything or just
+want to talk through something, you can reach us almost 24/7 on our Slack.
+To ask a question:
+
+1. Join our Slack: [https://slack.hackclub.io](https://slack.hackclub.io)
+2. Then join the `#curriculum` channel: [https://starthackclub.slack.com/messages/curriculum/](https://starthackclub.slack.com/messages/curriculum/)
+3. Ask your question!
+
+## How to Contribute to the Workshops
+
+There are three ways to contribute:
+
+1. [Create an Issue](#file-an-issue)
+2. [Improve an existing workshop](#improve-an-existing-workshop)
+3. [Create a new workshop](#create-a-new-workshop)
+
+### File an Issue
+
+If you notice any problems or have an idea you want to exist, you can create a
+GitHub issue describing the improvement you want to see. Click
+[here](https://github.com/hackclub/hackclub/issues/new) to create a new GitHub
+issue.
+
+### Improve an Existing Workshop
+
+If you want to go further, you can then submit a pull request with the changes
```

Comment:
actually decided not to add `first-timers-only` because I think the mentioning of this tag is less specific to workshops and would fit better in the root `CONTRIBUTING.md`